pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -470,5 +472,15 @@ func (c *schedulingController) createOrUpdatePlacementDecision(
 		"DecisionUpdate", "DecisionUpdated",
 		"Decision %s is updated with placement %s in namespace %s", placementDecision.Name, placement.Name, placement.Namespace)
 
+	// update the event with prioritizer score.
```

Comment:
I think what you have here should be each cluster's total score in the decision, right?

Reply:
yes, change to update cluster total score to event as below. 
```
Events:
  Type    Reason          Age   From                 Message
  ----    ------          ----  ----                 -------
  Normal  DecisionCreate  49m   placementController  Decision placement1-decision-1 is created with placement placement1 in namespace default
  Normal  DecisionUpdate  49m   placementController  Decision placement1-decision-1 is updated with placement placement1 in namespace default
  Normal  ScoreUpdate     49m   placementController  prod-9:100 prod-10:100 prod-6:100 prod-8:100 prod-7:100
```